src/averageAges.js

```diff
@@ -15,11 +15,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const ONE_CENTURY = 100;
+
+  const filteredByMans = people.filter(
+    ({ sex, died }) => century
+      ? sex === 'm' && Math.ceil(died / ONE_CENTURY) === century
```

Comment:
```suggestion
const MAN = 'm';
const isMan = sex => sex === MAN;

? isMan(sex) && Math.ceil(died / ONE_CENTURY) === century
```
* Avoid using magic strings - it's a programming antipattern. Consider using constants with meaningful names instead.
* Follow the DRY (don't repeat yourself) principle. Consider using helper functions for repeating functionality